src/lib/common/homestore_config.fbs

```diff
@@ -261,6 +261,9 @@ table Consensus {
 
     // Frequency to flush durable commit LSN in millis
     flush_durable_commit_interval_ms: uint64 = 500;
+
+    // Log difference to determine if the follower is in resync mode
+    resync_log_idx_threshold: int64 = 100;
```

Comment:
we might want to tune this number later with E2E env,  I am fine with this number for now.